package.xml

```diff
@@ -1,5 +1,6 @@
-<?xml version="1.0" ?>
-<package format="2">
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
   <name>planning_simulator_launcher</name>
   <version>0.0.0</version>
```

Comment:
IIRC 0.0.0 is not a valid version number for ROS packages, could you bump it to 0.0.1? Thanks.